src/containers/ItemDetailContainer.js

```diff
@@ -157,6 +160,32 @@ export class ItemDetailContainer extends Component {
     return itemResponse._source;
   }
 
+  populateGTMDataLayer(item) {
+    let environment = '';
+    const origin = window.location.origin;
+
+    if (origin.indexOf('rdc-staging') > -1) {
+      environment = 'staging';
+    } else if (origin.indexOf('dc.library.northwestern.edu') > -1) {
```

Comment:
Does this account for http://digitalcollections.library.northwestern.edu/ ?

Reply:
@kdid Ah, no it doesn't.  I didn't know that was a real url.  Do you know what else would be "production" urls?   Or staging urls I may be missing here?

Reply:
I'm not sure. I don't know of any others but maybe check with @davidschober